nbgrader/plugins/export.py

```diff
@@ -35,7 +41,24 @@ def export(self, gradebook):
         else:
             dest = self.to
 
+        if len(self.student) == 0:
+            allstudents = False
```

Comment:
```suggestion
            allstudents = []
```

`[]` evaluates to False in Python, and I think it's probably better to keep the datatypes consistent between different branches of the conditional.